rent_golem/backend/rent_golem/clusters/tests.py

```diff
@@ -1,3 +1,225 @@
-from django.test import TestCase
+from typing import List, Optional, Tuple
+from django.shortcuts import reverse
 
-# Create your tests here.
+from rest_framework.test import APIClient, APITestCase, APIRequestFactory
+from rest_framework import status
+
+from .models import Cluster
+from accounts.models import User
+
+
+class ClusterViewSetTest(APITestCase):
+    def setUp(self) -> None:
+        self.factory = APIRequestFactory()
+        self.client = APIClient()
+        self.owner = User.objects.create_user(email="test@test.com", password="test1234")
+        self.insignificant_user = User.objects.create_user(email="test2@test.com", password="test1234")
+
+    def create_cluster_object(
+            self,
+            uuid: str = "dcec5482-59ef-4c0e-9ea4-4e5451c3cbda",
+            package_type: str = Cluster.Package.AUTOMATIC,
+            status: str = Cluster.Status.STARTING,
+            additional_params=None,
+            size: int = 5,
+            owner: User = None
+    ) -> Cluster:
+        if owner is None:
+            owner = User.objects.create_user(email="test@test.com", password="test1234")
+        return Cluster.objects.create(
+            uuid=uuid,
+            package_type=package_type,
+            status=status,
+            additional_params=dict() if additional_params is None else additional_params,
+            size=size,
+            owner=owner
+        )
+
+    def test_post_one_object_to_db_as_registered_user(self):
+        cluster_data = {
+            "uuid": "aaec5482-59ef-4c0e-9ea4-4e5451c3cbda",
+            "package_type": Cluster.Package.AUTOMATIC,
+            "status": Cluster.Status.STARTING,
+            "additional_params": {},
+            "size": 5,
+            "owner": self.owner.id
+        }
+
+        jwt = self.client.post(
+            reverse("token_obtain_pair"), data={"email": self.owner.email, "password": "test1234"}
+        ).data["access"]
+
+        response = self.client.post(
+            reverse('cluster-list'), data=cluster_data, headers={"Authorization": f"Bearer {jwt}"}, format="json")
+
+        self.assertEquals(response.status_code, status.HTTP_201_CREATED)
+        cluster = Cluster.objects.get(uuid=cluster_data["uuid"])
+        self.assertEquals(cluster_data["package_type"], cluster.package_type)
+        self.assertEquals(cluster_data["status"], cluster.status)
+        self.assertEquals(cluster_data["additional_params"], cluster.additional_params)
+        self.assertEquals(cluster_data["size"], cluster.size)
+        self.assertEquals(cluster_data["owner"], self.owner.id)
+
+    def test_post_one_object_to_db_as_anonymous_user(self):
```

Comment:
Maybe `test_if_post_one_object_to_db_as_anonumous_will_fail` is better name. You can see what is tested and what should be the result.